inputTransformation.txt

```diff
@@ -0,0 +1,10 @@
+setGlobalScale=1
+setXScale=1
+setYScale=1
+setZScale=1
+setXRotation=0
+setYRotation=0
+setZRotation=0
+setXTranslation=0
+setYTranslation=0
+setZTranslation=0
```

Comment:
seems the shear groups were forgotten when i sent the sample so they could be added in here